recipes/cm-unicode/meta.yaml

```diff
@@ -0,0 +1,109 @@
+{% set name = "cm-unicode" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://kent.dl.sourceforge.net/project/{{ name }}/{{ name }}/{{ version }}/{{ name }}-{{ version }}-ttf.tar.xz
+  sha256: 2609c14450f42d0bcd40203900afcb1d693521a9b24a18c65e14b6b0585ff150
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
```

Comment:
You should probably just remove this section

Reply:
I tried that here - but it was not working:
https://github.com/conda-forge/staged-recipes/pull/8506/commits/d85a3b19de5df96978dfc33a0c4899b504ac0d28